fleet/lib/kyverno-conf/servicemonitor-kyverno-metrics.yaml

```diff
@@ -6,7 +6,6 @@ metadata:
   namespace: kyverno
   labels:
     o11y.eu/monitor: enabled
```

Comment:
I though the consensus was `lsst.io/monitor: "true"`? If so, please change.

Reply:
Updating the existing scrape labels is not the scope of this PR.  The goal is to decouple the prometheus CRDs from the prometheus deployment to resolve the circular dependency problem of the prometheus sts depending on `cm`s which come from the `snmp-exporter` deployment.  The string `kube-prometheus-stack` was removed where it wasn't required to make auditing the change easier. A double benefit to this change is allowing most other deployments to proceed even if the prometheus deployment hasn't completed.

All of these labels will need to be changed at the same time:

```
$ ack o11y.eu/monitor
strimzi-kafka-dashboards/fleet.yaml
6:  o11y.eu/monitor: enabled

mimir-pre/fleet.yaml
6:  o11y.eu/monitor: enabled

external-secrets/fleet.yaml
6:  o11y.eu/monitor: enabled

rook-ceph/fleet.yaml
6:  o11y.eu/monitor: enabled

kyverno-conf/servicemonitor-kyverno-metrics.yaml
8:    o11y.eu/monitor: enabled

kube-prometheus-stack/fleet.yaml
6:  o11y.eu/monitor: enabled

kyverno/fleet.yaml
7:  o11y.eu/monitor: enabled

mimir/overlays/ruka/values.yaml
11:      o11y.eu/monitor: enabled
17:      o11y.eu/monitor: enabled
22:      o11y.eu/monitor: enabled

mimir/overlays/pillan/values.yaml
11:      o11y.eu/monitor: enabled
17:      o11y.eu/monitor: enabled
22:      o11y.eu/monitor: enabled

mimir/fleet.yaml
7:  o11y.eu/monitor: enabled
```

Reply:
Also, this label will need to be updated on resource managed by argo and the person responsible is on vacation until next week.